diff --git a/dist/cli/build/utils/workerd.js b/dist/cli/build/utils/workerd.js
index 2567f7c3bb76d65b8703946e57c9d442814a2bc9..12c8e72a20964b0f181cfbdfe4167c3609bb17e0 100644
--- a/dist/cli/build/utils/workerd.js
+++ b/dist/cli/build/utils/workerd.js
@@ -43,7 +43,7 @@ export function transformBuildCondition(conditionMap, condition) {
 export function transformPackageJson(json) {
     const transformed = structuredClone(json);
     let hasBuildCondition = false;
-    if (json.exports) {
+    if (json.exports && typeof json.exports !== "string") {
         const exp = transformBuildCondition(json.exports, "workerd");
         transformed.exports = exp.transformedExports;
         hasBuildCondition ||= exp.hasBuildCondition;
